Cargo.toml

```diff
@@ -4,7 +4,7 @@ edition = "2021"
 publish = false
 
 [dependencies]
-bevy = "0.14.0"
+bevy = { git = "https://github.com/bevyengine/bevy/tree/latest"}
```

Comment:
This is latest release branch, not main branch.

Reply:
Ah I must've been on the wrong page when I copied it. Ill fix it.